unison-share-api/src/Unison/Server/Endpoints/NamespaceListing.hs

```diff
@@ -103,8 +106,7 @@ deriving instance ToSchema NamespaceObject
 
 data NamedNamespace = NamedNamespace
   { namespaceName :: UnisonName,
-    namespaceHash :: UnisonHash,
-    namespaceSize :: Size
```

Comment:
Does this need some corresponding change to the elm ui code?

Reply:
I asked Simon and he said the size is currently unused.